src/Token.php

```diff
@@ -26,6 +26,11 @@
     public function headers(): DataSet;
 
     /**
+     * Returns the token claims
+     */
+    public function claims(): DataSet;
```

Comment:
@Ocramius @lcobucci What you think about my previous question and about this PR?